app/views/full-page-examples/update-account-details/index.njk

```diff
@@ -0,0 +1,73 @@
+{# This example is not based on a live example #}
+{% extends "_generic.njk" %}
+
+{% from "back-link/macro.njk" import govukBackLink %}
+{% from "error-summary/macro.njk" import govukErrorSummary %}
+{% from "input/macro.njk" import govukInput %}
+{% from "date-input/macro.njk" import govukDateInput %}
+{% from "button/macro.njk" import govukButton %}
+
+{% set pageTitle = "Update account details" %}
+{% block pageTitle %}GOV.UK - {{ pageTitle }}{% endblock %}
+
+{% set mainClasses = "govuk-main-wrapper--l" %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+	text: "Back"
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+	<div class="govuk-grid-column-two-thirds">
+	  {% if errorSummary.length > 0 %}
+		{{ govukErrorSummary({
+		  titleText: "There is a problem",
+		  errorList: errorSummary 
+		}) }}
+	  {% endif %}
+	  <h1 class="govuk-heading-xl">{{ pageTitle }}</h1>
+
+	  <form method="post" novalidate>
+
+			{{ govukInput({
+				label: {
+					text: "Email address",
+					classes: "govuk-label--m"
+				},
+				type: "email",
+				id: "email",
+				name: "email",
+				value: values["email"],
+				errorMessage: errors["email"],
+				autocomplete: "email",
+				attributes: {
+					spellcheck: false
+				}
+			}) }}
+
+			{{ govukInput({
+				label: {
+					text: "Password",
```

Comment:
This is non-blocking but I wonder if it it would be clearer to the use what's expected if this said "New password". Maybe that's implicit in the heading text, I don't know. Feel free to ignore.

Reply:
Yeah fair that sounds good.